style.css

```diff
@@ -9,6 +9,11 @@ body {
 }
 
 @media only screen and (max-width: 550px) {
```

Comment:
- [ ] [OPTIONAL] I know this is for Mobile design, but it is advisable to build your website mobile-first. This means style the website for mobiles devices first, then use `min-width` to style the website for desktops or bigger devices.

Reply:
I will add that

Thanks

Reply:
You're welcome